extractor/src/main/kotlin/de/snowii/extractor/extractors/Blocks.kt

```diff
@@ -52,9 +52,7 @@ class Blocks : Extractor.Extractor {
             val statesJson = JsonArray()
             for (state in block.stateManager.states) {
                 val stateJson = JsonObject()
-                val id = stateIdCounter
-                stateIdCounter++
-                stateJson.addProperty("id", id)
+                stateJson.addProperty("id", Block.getRawIdFromState(state))
```

Comment:
I think you can then remove the id counter vars right ?

Reply:
I'm not sure what you mean, they are removed arent they?